openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -190,9 +190,10 @@ def add_field(
                 See `ALLOWED_RESTRICTIONS` for acceptable values.
 
             options (list): For "select" fields, a list of tuples
-                (value, display_name) representing the options available to
+                (value, display_name, default) representing the options available to
```

Comment:
Hmm, I see. It's funny I changed this comment when in the test I wrote I didn't even do it this way :X